upp-msk-provisioner/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN apk --update --no-cache add \
     openssl-dev \
     build-base \
   && pip3 install --upgrade pip \
+  && pip3 install cryptography==2.9.2 \
```

Comment:
Non-blocking: I think it's going to be more consistent if the **cryptography** version is extracted as an ENV variable like Ansible and Kubectl ones